include/circt/Dialect/Calyx/CalyxControl.td

```diff
@@ -372,3 +372,38 @@ def StaticRepeatOp : CalyxContainer<"static_repeat", [
   ];
 
 }
+
+def RepeatOp : CalyxContainer<"repeat", [
+    ControlLike
+  ]> {
+  let summary = "Calyx Dynamic Repeat";
+  let arguments = (ins
+    I32Attr:$count
+  );
+  let description = [{
+    The "calyx.repeat" operation represents the repeated execution of
+    the control within its region.
```

Comment:
Can you spell out what is dynamic in this, i.e., the body? This tripped me up :-)

Reply:
Ah yeah I see how that can be confusing, just fixed it.